app/views/rent_mailer/rent_created.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1 style='color: green'>A new rent was created!</h1>
+
+<p>
+  <h2>Dear customer <b style='color: blue'><%= @user_renting.first_name+' '+@user_renting.last_name %>,</b></h2>
```

Comment:
For string interpolation we tend to prefer this syntax `"#{@rent.user.first_name} #{@rent.user.last_name}"`